malaria-app-android/src/main/java/com/peacecorps/malaria/ui/user_profile/UserProfileFragment.java

```diff
@@ -38,11 +38,8 @@
 
     private UserProfilePresenter<UserProfileFragment> presenter;
 
+    // returns a new instance of the Fragment
     public static UserProfileFragment newInstance() {
-//        Bundle args = new Bundle();
```

Comment:
Remove commented code.

Reply:
Since I'm not passing any values between activity-fragment using this method, I removed this method locally for the next PR. For current usage, there is no need of newInstance(), calling new UserProfileFragment() in main activity works fine :+1:

Reply:
Alright.